docker/official/remco/templates/jaas-loginmodule.conf

```diff
@@ -22,36 +22,36 @@
         rolePrefix="{{ getv("/rundeck/jaas/ldap/roleprefix", "") }}"
         cacheDurationMillis="{{ getv("/rundeck/jaas/ldap/cachedurationmillis", "600000") }}"
         reportStatistics="{{ getv("/rundeck/jaas/ldap/reportstatistics", "true") }}"
-    {% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") -%}
+{% if exists("/rundeck/jaas/ldap/roleusernamememberattribute") %}
```

Comment:
Testing this it appears the spacing in the if/else statements does not impact the rendered output(tested with remco master and the pinned version). Are you seeing something different?

Reply:
It's fine individually, but when I apply multiple settings the whitespace stripping at the end of the if can cause them to end up clumped up at the start of the line.

So setting:
```
RUNDECK_JAAS_LDAP_ROLEMEMBERATTRIBUTE=member
RUNDECK_JAAS_LDAP_ROLEOBJECTCLASS=groupOfNames
RUNDECK_JAAS_LDAP_SUPPLEMENTALROLES=user
```

Results in:
```
        cacheDurationMillis="600000"
        reportStatistics="true"
userLastNameAttribute=sn
userFirstNameAttribute=givenName
userEmailAttribute=mail
```
being rendered in the template. This doesn't break rundeck starting but does break those attributes.